src/vss_tools/vspec/vssexporters/vss2samm/helpers/ttl_helper.py

```diff
@@ -148,7 +148,7 @@ def handle_branch_node(
     for child_node in vss_node.children:
         child_node_uri = None
 
-        if split_vss and child_node.depth <= cfg.SPLIT_DEPTH and child_node.data.type is NodeType.BRANCH:
+        if split_vss and child_node.depth <= Config.SPLIT_DEPTH and child_node.data.type is NodeType.BRANCH:
```

Comment:
At this stage, all of OUTPUT_NAMESPACE, VSPEC_VERSION and SPLIT_DEPTH are initialized i.e., they have values other than Null.